Controllers/TodoController.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Efcore_demo.Data;
+using Efcore_demo.Models;
+using Efcore_demo.Repositories.Interfaces;
+using Efcore_demo.Services.Interfaces;
+
+namespace TodoApplication.Controllers;
+
+public class TodoController : Controller
+{
+    public TodoController(
+       ITodoService todoService,
+       ITodoRepository todoRepository
+    )
+    {
+        _todoService = todoService;
+        _todoRepository = todoRepository;
+
+    }
+    public ITodoService _todoService { get; }
+    public ITodoRepository _todoRepository { get; }
+
+    public IActionResult Index()
```

Comment:
use `Index` rather then `GoTodos` follow the standard apporach

Reply:
hunxa